src/components/place-card/place-card.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const placeCard = (props) => {
```

Comment:
Я писал, что все компоненты в реакте должны быть с большой буквы!
```suggestion
const PlaceCard = (props) => {
```